cmd/create.go

```diff
@@ -1455,46 +1456,78 @@ Por padrão, o deployment embutido no binário é utilizado.`,
 				fmt.Println("\nE inicie o serviço:")
 				fmt.Println("   sudo systemctl enable docker")
 				fmt.Println("   sudo systemctl start docker")
+			}
+			if runtime.GOOS == "darwin" && containerEngine == "podman" {
+				// macOS podman
+				fmt.Println("\n📦 Para macOS, recomendamos Podman:")
+				fmt.Println("1. Instale o Homebrew caso não tenha:")
+				fmt.Println("   /bin/bash -c \"$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)\"")
+				fmt.Println("2. Instale o Podman")
+				fmt.Println("   brew install podman")
+				fmt.Println("3. Inicie o Podman:")
+				fmt.Println("   podman machine init")
+				fmt.Println("   podman machine start")
+			} else if runtime.GOOS == "linux" && containerEngine == "podman" {
+				// Linux podman
+				fmt.Println("\n📦 Para Linux, use instruções oficiais de instalação para seu sistema operacional:")
+				fmt.Println("   https://podman.io/docs/installation#linux-distributions")
+				fmt.Println("\nE inicie o serviço:")
+				fmt.Println("   sudo systemctl enable podman")
+				fmt.Println("   sudo systemctl start start")
```

Comment:
O comando não seria para inicialização do podman?

```suggestion
				fmt.Println("   sudo systemctl start podman")
```

Reply:
Opa, arrumei uma coisa e mandei um erro :) 

Vou mandar o patch aqui